src/parser.mly

```diff
@@ -54,8 +54,8 @@
 
```

Comment:
If we're going to get rid of all references to Disjunctions remember to remove the SEMICOLON token above.